### PR TITLE
25 landing page

### PR DESCRIPTION
**Pull request by mateusmorgado**

### CODE REVIEW

**Comment by mateusmorgado** [comment]
I did finish the landing page. buttons still need a path to Sign up and to Login pages

File: package-lock.json
Comment (verascheunert) [inline package-lock.json:1]: Please delete the package-lock file, we want to keep only the pnpm-lock.yaml :)

**Review by verascheunert — COMMENTED** [review]
Very nice work, I left just a few small remarks and one question :)

File: app/page.tsx
Comment (mateusmorgado) [reply app/page.tsx:37]: I have separated the elements into three groups, so when I apply the flex-col property I can use space-between to make the page responsive.

**Review by mateusmorgado — COMMENTED** [review]
I did answer Vera's comments. waiting for feedback/pull request

**Comment by verascheunert** [comment]
Awesome thanks for the changes! Can you reverse merge main into your branch, after that we can merge it :)

**Review by RealBirdMan91 — APPROVED** [review]

**Review by RealBirdMan91 — APPROVED** [review]

**Comment by RealBirdMan91** [comment]
there is still some conflict in globals.css please fix first

**Comment by mateusmorgado** [comment]
Hey @verascheunert, I did put the landing page where it should be. Could you pl review this pull request?

**Comment by mateusmorgado** [comment]
@verascheunert / @RealBirdMan91 now this is ready for merging. pl accept

**Pull request merged**
